### ISSUE TITLE
Vega 64 mV and clocks at stages

### ISSUE THREAD

**Issue by sklorpion** [amezin/amdgpu-tweakd#1]
Hi,

is it possible to undervolt via amdgpu-teaked? cant find that option. I would really like to make it possible to play with stage volting and speed, maybe it's even possible to change mems?

good job.

**Comment by amezin** [comment]
No, it's not currently possible. I have plans to add this feature, but no time to work on the project currently :(
